static/sass/_pattern_navigation.scss

```diff
@@ -0,0 +1,70 @@
+
+@mixin canonical-p-navigation {
+
+// Style for main navigation
+.p-navigation__dropdown a.is-selected {
+    border-left: 4px solid white;
+    color: white !important;
+  }
+}
+
+// Meganav
+.slide-animation {
```

Comment:
This is probably the best thing I've done in this feature. It does this slight animation, which makes it look cool, but it's actually because we are making a call to greenhouse, and it goes down quite frequently according to sentry, so making a JS request should protect the site from going down (navigation is loaded everywhere)